doc/users/next_whats_new/selector_improvement.rst

```diff
@@ -0,0 +1,8 @@
+Selectors improvement
+---------------------
+
+The `~matplotlib.widgets.RectangleSelector` and
+`~matplotlib.widgets.EllipseSelector` can now be rotated interactively. The rotation is
+enabled and disabled by pressing the 'r' key.
```

Comment:
That's a bit ambiguous. Is it one key press (plus release) or do you have to press and hold?